### PR TITLE
change location attr to dict

### PR DESCRIPTION
**Pull request by akkaky**
Location attribute changed from string to dict. This allows the GPS coordinates to be used as the name of the place and to replace the coordinates with the preferred name.
Also returned the json field to `defaults` for correct display in the documentation.

### CODE REVIEW

**Comment by tych0** [comment]
Can you separate these two things into two logical commits, and include your reasoning for the change in the PR notes in the commit message body as well?

**Comment by akkaky** [comment]
Ok! Will make new PR's.

**Comment by akkaky** [comment]
Have I to close this PR?

**Comment by tych0** [comment]
No, you can just force push with `git push -f` to this branch when you've updated the commits. Thanks!

**Comment by akkaky** [comment]
Like this or wrong?

**Comment by tych0** [comment]
That looks good to me, but can you add your reasoning about *why* you changed the list to a dict in that commit? e.g.:

> This allows the GPS coordinates to be used as the name of the place and to replace the coordinates with the preferred name.

**Comment by akkaky** [comment]
Add to commit text? Sorry, I just study work with git.

File: libqtile/widget/wttr.py [hunk 1]
```diff
@@ -5,9 +5,12 @@
 
 class Wttr(GenPollUrl):
     """Display weather widget provided by wttr.in_.
+
```
Comment (tych0) [inline libqtile/widget/wttr.py]: It looks like this is a commit full of whitespace only changes; if you want these can you squash them (and the message) into the previous commit?

**Comment by akkaky** [comment]
@tych0 Thanks for the advice. Now I figured out how to make clear commits. Before I made a scrapyard.

**Comment by tych0** [comment]
Sure thing, thanks for the patches!

**Pull request merged**